File: backend/stove-app-parent/src/main/java/by/shumpanov/stove/stove_app_parent/dto/RegisterRequest.java
package by.shumpanov.stove.stove_app_parent.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class RegisterRequest {
    @JsonProperty("email")
    private String email;
    @JsonProperty("password")
    private String password;
    @JsonProperty("phone_number")
    private String phoneNumber;
    @JsonProperty("full_name")
    private String fullName;

}

File: android/app/src/main/java/com/example/stove/data/dto/RegisterRequest.kt
package com.example.stove.data.dto

import com.google.gson.annotations.SerializedName

data class RegisterRequest(
    @SerializedName("full_name")
    val fullName: String,
    @SerializedName("phone_number")
    val phoneNumber: String,
    val email: String,
    val password: String
)
